The error is related to `account_invoice_facturx` but I don't know it and I don't know exactly how to solve it.

```
2024-12-03 13:15:12,935 368 ERROR odoo odoo.addons.account_invoice_facturx.tests.test_facturx_invoice: ERROR: TestFacturXInvoice.test_deep_customer_invoice
Traceback (most recent call last):
  File "/__w/edi/edi/account_invoice_facturx/models/account_move.py", line 873, in generate_facturx_xml
    xml_check_xsd(xml_byte, flavor="factur-x", level=ns["level"])
  File "/opt/odoo-venv/lib/python3.8/site-packages/facturx/facturx.py", line 172, in xml_check_xsd
    xsd_etree_obj = etree.parse(importlib.resources.files(__package__)
AttributeError: module 'importlib.resources' has no attribute 'files'
```

--- a/account_invoice_facturx/__manifest__.py
+++ b/account_invoice_facturx/__manifest__.py
@@ -17,7 +17,7 @@
         "base_facturx",
         "base_vat",
     ],
-    "external_dependencies": {"python": ["factur-x"]},
+    "external_dependencies": {"python": ["factur-x<=3.1"]},
     "data": [
         "views/res_partner.xml",
         "views/res_config_settings.xml",

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,5 +1,5 @@
 # generated from manifests external_dependencies
-factur-x
+factur-x<=3.1
 pypdf
 PyPDF2
 PyYAML

